package.json

```diff
@@ -41,7 +41,7 @@
   },
   "devDependencies": {
     "@types/jest": "^26.0.15",
-    "@types/node": "^14.14.9",
+    "@types/node": "^15.6.1",
```

Comment:
What about users with M1 Apple Silicon, for whom Node 15 was broken (cc @thimy  👋  )

Reply:
Node 15 still works!

Reply:
This package is the bundle of Types for typescript, but it doesn't prevent the code to be Node 14 compatible. It only gets the latest updates on types.

I'm pretty sure it will work on M1 computers :)